monkey/monkey_island/cc/next_ui/src/assets/svg-components/MonkeyHead.svg.tsx

```diff
@@ -1,11 +1,21 @@
-const MonkeyHeadSvg = () => (
+interface SvgProps {
+    className?: string;
+    color?: string;
+    height?: string;
+}
+
+const MonkeyHeadSvg = (props: SvgProps) => (
```

Comment:
We should call it MonkeyIcon, AppIcon or something like that. Also, it should take the color from the theme by default, I'm not sure what "currentColor" does

Reply:
If my understanding is correct, `currentColor` is the value of the `color` property